### PR TITLE
Feat/react basic

### PR DESCRIPTION
**Pull request by dirkmc**
Fixes a couple of bugs, and allows directory navigation and viewing files

screen shot 2018-03-22 at 5 38 42 pm

screen shot 2018-03-22 at 5 38 54 pm

### CODE REVIEW

File: src/styles/css/main.css [hunk 1]
```diff
@@ -3743,10 +3743,10 @@ code, .code {
   width: 100%; }
 
 .w-third {
-  width: 33.3333333333%; }
```
Comment (magik6k) [inline src/styles/css/main.css]: I'm pretty sure this file shouldn't be committed to git.

**Comment by magik6k** [comment]
You should be able to add syntax highlighting fairly easily with highlight.js - https://highlightjs.org/usage/ - https://github.com/ipfs-shipyard/IGiS/blob/master/src/main/scala/igis/app/controllers/BlobController.scala#L43

**Comment by magik6k** [comment]
btw, Can you open this pr to the feat/react branch? I'll switch them once the react version has more of the features from the scala one

**Comment by dirkmc** [comment]
Oops yes sorry I meant to open the PR against the feat/react branch, I'll close this one and resubmit

**Pull request closed without merging**